openmc/deplete/results_list.py

```diff
@@ -296,3 +298,72 @@ def get_step_where(
             "relative tolerances {} and {}.".format(
                 time, time_units, atol, rtol)
         )
+
+    def export_to_materials(self, burnup_index, nuc_with_data=None):
+        """Return openmc.Materials created from these depletion results.
+
+        Parameters
+        ----------
+        burn_index : int
+            Index of burnup step to evaluate
+        nuc_with_data : Iterable of str, optional
+            Nuclides iterable to evaluate with neutron data.
+            This must be specified if not all nuclides appearing in
```

Comment:
"can be specified" would be more accurate since the default behavior is probably the common case

Reply:
Sounds good!